nima_core/precognition.py

```
@@ -154,41 +155,98 @@ def _get_conn(self) -> sqlite3.Connection:
+            rows = src.execute(
+                """
+                SELECT timestamp, who, text, summary, themes
+                FROM memory_nodes
+                WHERE is_ghost = 0
+                  AND timestamp >= ?
+                  AND layer IN ('input', 'contemplation', 'output', 'episodic', 'semantic')
+                ORDER BY timestamp ASC
+                """,
+                (cutoff_ms,),
+            ).fetchall()
         except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)